content/en/tracing/guide/instrument_custom_method.md

```diff
@@ -0,0 +1,177 @@
+
+Spans are the building blocks of APM Traces. You can instrument entire methods in your code. Follow the code sample below to create a new span that traces your method and includes it within your traces.
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+```java
+TBD - Java
+```
+
+{{% /tab %}}
+{{% tab "Python" %}}
+
+```python
+TBD - Python
+```
+
+{{% /tab %}}
+{{% tab "Ruby" %}}
+
+```ruby
+TBD - Ruby
+```
+
+{{% /tab %}}
+{{% tab "Go" %}}
+
+
+```go
+TBD - Go
+```
+
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+```javascript
+TBD - js
+```
+
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+```csharp
+TBD - C#
+```
+
+{{% /tab %}}
+{{% tab "PHP" %}}
+
+```php
+TBD - PHP
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Instrument individual componenets of your methods to control the precision of your spans
+
+**Follow the example to get your code instrumented**.
+
+In this example walks through adding child spans to the `BackupLedger.write` span created above. This method adds a row to the transaction ledger, saves the entire ledger in memory, and then notifies the service that the ledger is saved. This examples adds spans to track each of the three actions.
+
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+```java
+TBD - Java
+```
+
+{{% /tab %}}
+{{% tab "Python" %}}
+
+```python
+TBD - Python
+```
+
+{{% /tab %}}
+{{% tab "Ruby" %}}
+
+```ruby
+TBD - Ruby
+```
+
+{{% /tab %}}
+{{% tab "Go" %}}
+
+```go
+TBD - Go
+```
+
+{{% /tab %}}
+{{% tab "Node.js" %}}
+
+```javascript
+TBD - js
+```
+
+{{% /tab %}}
+{{% tab ".NET" %}}
+
+```csharp
+TBD - C#
+```
+
+{{% /tab %}}
+{{% tab "PHP" %}}
+
+```php
+TBD - PHP
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Leverage the Datadog UI to see your new custom spans
+
+Now that you have instrumented your business logic, it's time to see the results in the Datadog APM UI.
+
+1. **Go to the [Service List][1]**, identify the service you added custom spans to and **go to the Service Page**, from there, **click on the specific resource** you worked on. Change the time filter to `The past 15 minutes` and scroll down to the Span Summary table:
+
+    {{< img src="tracing/guide/custom_span/custom_span_3.png" alt="Span Summary Table" responsive="true" style="width:90%;">}}
+
+    *You should now be able to find the new spans you've added*
+
+    The Span Summary table is a great way to have an at-a-glance view of aggregate information about the spans that make up your traces. Here you can identify spans that repeat an abnormal amount of times indicating some looping or database access inefficiency (like the `n+1` issue).
+
+2. **Scroll down to the Traces list** and click into one of the traces you see there.
+
+    {{< img src="tracing/guide/custom_span/custom_span_4.png" alt="Analytics View" responsive="true" style="width:90%;">}}
```

Comment:
I think we should zoom the image a bit, in a way that operation names are visible (the ones we'll define in the code snippets).

Reply:
@palazzem Ok, I'll have to photoshop those in, so can you let me know what those will be and we'll do that when the guide is ready?